worker/backup_ee.go

```diff
@@ -188,7 +192,7 @@ func ProcessBackupRequest(ctx context.Context, req *pb.BackupRequest, forceFull
 		}
 	}
 
-	m := Manifest{Since: req.ReadTs, Groups: predMap}
+	m := Manifest{Since: req.ReadTs, Groups: predMap, DropOperations: dropOperations}
```

Comment:
Should you append the new `dropOperations` to the existing backups? What if you had incremental backups?

Reply:
No, each backup whether full or incremental creates a new manifest file. So, here it is writing only to the new manifest file, no old files are modified.